src/modules/formlyBootstrap.js

```diff
@@ -70,8 +72,19 @@ angular.module('formlyBootstrap', ['formly'], function configFormlyVanilla(forml
     }
   });
 
+
   function getFieldTemplateUrl(name) {
     return 'fields/formly-field-' + name + '.html';
   }
 
 });
+
+angular.module('formlyBootstrap').run(function (formlyConfig, $http, $templateCache) {
+  formlyConfig.templateManipulators.preWrapper.push(function (template, options) {
+    if (options.type !== 'input' || (!options.templateOptions.addonLeft && !options.templateOptions.addonRight)) {
+      return template;
+    }
+    var tmpl = $templateCache.get('other/formly-other-bootstrap-addons.html');
```

Comment:
I would recommend that you still use `$http` because during local development the template wont be in the `$templateCache` and you'll need to get it via `$http`. By specifying `cache: $templateCache`, `$http` will lookup the url in the cache before making a request so when people are actually using the library they'll never make an http request. So anyway, I'd delete this code and uncomment the lines below.